src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -334,9 +334,15 @@ parseObjectLiteral :: TokenParser Expr
 parseObjectLiteral = ObjectConstructor <$> braces (commaSep parseIdentifierAndValue)
 
 parseIdentifierAndValue :: TokenParser (String, Maybe Expr)
-parseIdentifierAndValue = (,) <$> (C.indented *> (lname <|> stringLiteral) <* C.indented <* colon)
-                              <*> (C.indented *> val)
+parseIdentifierAndValue =
+  do
+    name <- ident
+    b <- P.option (Just $ Var $ (Qualified Nothing) $ (Ident name)) rest
+    return (name, b)
+  <|> (,) <$> ident <*> rest
```

Comment:
One branch should be `stringLiteral` (with a required value) and the other should be an `lname` (without) since `Var` needs to take an `lname` not an arbitrary string.